components/script/dom/element.rs

```diff
@@ -693,6 +694,30 @@ impl LayoutElementHelpers for LayoutJS<Element> {
         }
     }
 
+    #[allow(unsafe_code)]
+    fn get_lang_for_layout(&self) -> String {
+        unsafe {
+            let mut current_node = Some(self.upcast::<Node>());
```

Comment:
Oh, ok... I'm still kinda uneasy with the `None => continue`, but I can probably live with it?